aodntools/timeseries_products/hourly_timeseries_template.json

```diff
@@ -0,0 +1,377 @@
+            "units": "dbar",
+            "valid_max": 12000.0,
+            "valid_min": -15.0,
+            "_FillValue": 999999.0
+            },
+        "PRES_REL_quality_control": {
+            "flag_meanings": "No_QC_performed Good_data Probably_good_data Bad_data_that_are_potentially_correctable Bad_data Value_changed Not_used Not_used Not_used Missing_value",
+            "flag_values": "0, 1, 2, 3, 4, 5, 6, 7, 8, 9",
+            "long_name": "sea_water_pressure_due_to_sea_water status_flag",
+            "standard_name": "depth status_flag",
+            "_FillValue": 99
+            },
+        "instrument_index": {
+            "long_name": "which instrument this obs is for",
+            "instance_dimension": "INSTRUMENT"
+            },
+        "instrument_id": {
+            "long_name": "source deployment code, instrument make, model, serial_number"
+            },
+        "source_file": {
+            "long_name": "source file for this instrument"
+            },
+        "TEMP": {
+            "ancillary_variables": "TEMP_quality_control",
+            "_FillValue": 999999.0,
+            "long_name": "sea_water_temperature",
+            "standard_name": "sea_water_temperature",
+            "units": "degrees_Celsius",
+            "valid_max": 40.0,
+            "valid_min": -2.5
+            },
+        "TEMP_quality_control": {
+            "_Unsigned": "false",
+            "_FillValue": 99,
+            "flag_meanings": "No_QC_performed Good_data Probably_good_data Bad_data_that_are_potentially_correctable Bad_data Value_changed Not_used Not_used Not_used Missing_value",
+            "flag_values": "0, 1, 2, 3, 4, 5, 6, 7, 8, 9",
+            "long_name": "quality flag for sea_water_temperature",
+            "standard_name": "sea_water_temperature status_flag"
+        },
+        "PSAL": {
+            "ancillary_variables": "PSAL_quality_control",
+            "_FillValue": 999999.0,
+            "long_name": "sea_water_practical_salinity",
+            "standard_name": "sea_water_practical_salinity",
+            "units": "S m-1",
+            "valid_max": 41.0,
+            "valid_min": 2.0
+            },
+        "PSAL_quality_control": {
+           "_Unsigned": "false",
+            "_FillValue": 99,
+            "flag_meanings": "No_QC_performed Good_data Probably_good_data Bad_data_that_are_potentially_correctable Bad_data Value_changed Not_used Not_used Not_used Missing_value",
+            "flag_values": "0, 1, 2, 3, 4, 5, 6, 7, 8, 9",
+            "long_name": "quality flag for sea_water_practical_salinity",
+            "standard_name": "sea_water_practical_salinity status_flag"
+            },
+        "CNDC": {
+            "ancillary_variables": "CNDC_quality_control",
+            "_FillValue": 999999.0,
+            "long_name": "sea_water_electrical_conductivity",
+            "standard_name": "sea_water_electrical_conductivity",
+            "units": "1",
```

Comment:
Do we need to include conductivity (`CNDC`)?
I didn't think so, but if we do, the compliance checker says its units (based on the standard_name) should be "S m-1" or equivalent.

Reply:
Fixed